demo/server/src/modules/index.ts

```diff
@@ -0,0 +1,39 @@
+import { Auth } from '@raincatcher/auth-passport';
+import { BunyanLogger, Logger } from '@raincatcher/logger';
+import * as express from 'express';
+import { connect as syncConnector } from './datasync/Connector';
+import { router as syncRouter } from './datasync/Router';
+import { init as authInit } from './passport-auth';
+
+export let securityMiddleware: Auth;
+export let logger: Logger;
+
+// Setup all modules
+export function setupModules(app: express.Express) {
+  loggerSetup();
+  syncSetup(app);
+  securitySetup(app);
+}
+
+function loggerSetup() {
+  logger = new BunyanLogger({ name: 'Demo Application' });
+}
+
+function securitySetup(app: express.Express) {
+  setupPassportSecurity(app);
+}
+
+function setupPassportSecurity(app: express.Express) {
```

Comment:
[Question] Why can't this be called directly instead of having to go through securitySetup?